### PR TITLE
ACC-2127: Update host url to fetch vault value

### PR DESCRIPTION
**Pull request by Nazehs**
The Salesforce team want to test some changes they are making to the live chat in their staging environment, so they’d like the option to toggle using their staging or production environment.

This PR allows the consuming apps to use `liveChatStaging`  flag  and `LIVE_CHAT_STAGING_HOST` and `LIVE_CHAT_PROD_HOST` with their staging and prod url as the value in vault  for the consuming app to allow saleforce team to be able to use a toggler to change the url for the live chat.

This PR also contain some of the package that their versions were bumped

[Jira Ticket](https://financialtimes.atlassian.net/browse/ACC-2127)

### CODE REVIEW

File: demos/app.js [hunk 1]
```diff
@@ -25,16 +26,22 @@ const salesforceConfig = {
...
+	salesforceConfig.liveChatURL= liveChatStaging ? process.env.LIVE_CHAT_STAGING_HOST : process.env.LIVE_CHAT_PROD_HOST;
 	res.render('demo.jsx', {
 		style: 'popup',
 		salesforceConfig
 	});
 });
 
 app.get('/inline', (req, res) => {
+	const { liveChatStaging } = res.locals.flags;
+	console.log(liveChatStaging);
```
Comment (asugar13) [inline demos/app.js:43]: **Comment**: There's a console.log here. Did you make sure to run linting before pushing?

File: demos/client/demo.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import { LiveChatInline, LiveChatPopup } from '../../dist/browser';
 
 if(document.querySelector('[data-live-chat-style="inline"]')) {
-	new LiveChatInline().init(null, { demoMode: 'online', chatterBox: true });
+	new LiveChatInline().init(null, { demoMode: 'online', chatterBox: true, liveChatURL: 'https://ip-chatterbox-client-staging.herokuapp.com' });
```
Comment (asugar13) [inline demos/client/demo.js:4]: **Question**: Why is one of these links for staging and the other for prod?

**Review by asugar13 — COMMENTED** [review]
Left two questions

File: demos/app.js (hunk 1 above)
Comment (Nazehs) [reply demos/app.js:43]: [Comment] There is no linting rules for this package. That is why the console.log sneaked in successfully

File: demos/client/demo.js (hunk 2 above)
Comment (Nazehs) [reply demos/client/demo.js:4]: [Comment] it's a demo so i decided to use the 2 end points both prod and staging

Comment (asugar13) [reply demos/client/demo.js:4]: Can you add a comment in the README perhaps explaining this so it's more obvious when a developer picks this up?

**Review by asugar13 — DISMISSED** [review]
looks good!

**Review by asugar13 — APPROVED** [review]

**Pull request merged**